testResources/raml/custom-field/custom-field.raml

```diff
@@ -0,0 +1,10 @@
+#%RAML 1.0
+
+title: Shop API
+x-salesforce-sdk-class-name: Custom Shop API Name
```

Comment:
In RAML this would be `(salesforce-sdk-class-name)`. https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/#annotations
as opposed to https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/#x-other

and what if you followed our generator convention of:
```
(commerce-sdk):
  class-name: Custom Shop API Name
```

Reply:
Done! You can see this in the example test file `testResources/raml/custom-field/custom-field.raml`